### PR TITLE
[Distributed] Try to make test for null sourcefile

### PR DESCRIPTION
**Pull request by ktoso**
The test does not trigger the "source file is null" branch though, I'm trying to find out how to test this.

### CODE REVIEW

**Comment by ktoso** [comment]
refs https://github.com/apple/swift/pull/66842

File: test/ModuleInterface/SomeDistributedActor.swiftinterface [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// swift-interface-format-version: 1.0
+// swift-module-flags: -module-name SomeDistributedActor
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t/DistributedImport.swiftmodule -module-name DistributedImport %S/Inputs/someDistributedActorModule.swift
+// RUN: %target-swift-frontend -compile-module-from-interface -o %t/SomeDistributedActor.swiftmodule %s -I %t -verify
```
Comment (xedin) [inline test/ModuleInterface/SomeDistributedActor.swiftinterface:6]: Regardless of `source null` issue I think it's good to have a round-trip test like we have for property wrappers - https://github.com/apple/swift/blob/main/test/ModuleInterface/property_wrappers.swift. It emits swiftinterface, type-checks it and uses it to emit a module.

**Comment by ktoso** [comment]
I did a roundtrip test, does this look good @xedin ?

**Comment by ktoso** [comment]
@swift-ci please smoke test

**Pull request merged**